### ISSUE TITLE
Write CWL and Docker for bwa mem

### ISSUE THREAD

**Issue by rhshah** [msk-access/cwl-commandlinetools#50]
Tool: https://github.com/lh3/bwa
Version: v0.7.17

Example Command:

```bash
/opt/common/CentOS_7/bwa/bwa-0.7.17/bwa mem -K 1000000 -T 30 -t 8 /work/access/production/resources/reference/versions/hg19/Homo_sapiens_assembly19.fasta /juno/work/bergerm1/bergerlab/arorak/Project_benchmark_fgbio/FqToBamFgbio_2020-06-10/NovaSeq_Plasma_Healthy_Donor/DONOR11-N/DONOR11-N.fqtobam.clip.fastq.gz
```

- [x] CWL
- [x] Docker
- [x] Example Yaml
- [x] Readme
- [x] test on toil and cwltool

**Comment by rhshah** [comment]
@ionox0  want to make sure all the necessary things are done

**Comment by rhshah** [comment]
#42 Resolves #50
